frontend/gulpfile.js

```diff
@@ -345,8 +345,8 @@ function dist(ext, name, opt) {
     .pipe(opt.ngAnnotate ? g.rename : noop, name + '.annotated.' + ext)
     .pipe(opt.ngAnnotate ? gulp.dest : noop, './dist')
     .pipe(ext === 'js' ? g.uglify : g.minifyCss)
-    .pipe(g.rename, name + '.min.' + ext)
-    .pipe(gulp.dest, './dist')()
+    .pipe(g.rename, name + '.min.' + ext)()
+    //.pipe(gulp.dest, './dist')()
```

Comment:
Hmm, does this work? What I see is that minified files aren't moved to ```dist``` folder anymore.